Added EU status page link (#228)
mulesoft/docs-runtime-manager: modules/ROOT/pages/maintenance-and-upgrade-policy.adoc

> MuleSoft's policy is to perform all maintenance transparently and without any downtime to your integration(s) running in the worker cloud wherever possible. In cases where downtime for the console, platform services, or workers is unavoidable, MuleSoft communicates these maintenance windows at least eight hours in advance and typically seven days in advance, with the exception of critical security patches which may be fixed immediately.

Reviewer: ```suggestion
MuleSoft policy is to perform all maintenance transparently and without any downtime to your integrations running in the worker cloud wherever possible. In cases where downtime for the console, platform services, or workers is unavoidable, MuleSoft communicates these maintenance windows at least eight hours in advance and typically seven days in advance, with the exception of critical security patches, which might be fixed immediately.
```

> View the live status and detailed service history for the Runtime Manager console, CloudHub platform services, and the CloudHub worker cloud:

Reviewer: ```suggestion
View the current status and detailed service history for the Runtime Manager console, CloudHub platform services, and the CloudHub worker cloud:
```
